### PR TITLE
LRD-0001 Fix country combobox in city search page

### PR DESCRIPTION
**Pull request by sylee1245**
### Motivation
- country combo change event bug
### Modifications
- Fix country combobox in city
### 
LRD-0001

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # vue-news
 vue3 toy project
+Add router
+Add watch
```
Comment (jinsukLee) [inline README.md:4]: How about remove `watch`
```java
public static getMethod(){
log.error("")
}
```

**Review by jinsukLee — COMMENTED** [review]
Please check my comment~

File: README.md (hunk 1 above)
Comment (sylee1245) [reply README.md]: Thank you, I'll remove it

**Review by jinsukLee — APPROVED** [review]
LGTM 👍

**Pull request merged**